compile.bat

```diff
@@ -0,0 +1 @@
+"C:\Program Files\Java\jdk1.8.0_151\bin\javac" GreetingsProgram.java
```

Comment:
I don't like absolute path to Java which could be different on different computers